src/yenc.cc

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright 2007-2023 The SABnzbd-Team <team@sabnzbd.org>
+ * Copyright 2003, 2011 Alessandro Duca <alessandro.duca@gmail.com>
```

Comment:
All code from the original `yenc` is no longer present, so we can remove this and any references to Alessandro (I think).